### ISSUE TITLE
Windows machine needed

### ISSUE THREAD

**Issue by irinarada** [AdoptOpenJDK/openjdk-infrastructure#334]
@tellison kindly offered to help. 
@sxa555 and @smlambert can you add a description of the machines?
Shelley mentioned the build machines description might be enough.

**Comment by tellison** [comment]
Sorry I mis-spoke
> That windows box is a bare metal server ...

The build machines `build-win2012-1.adoptopenjdk.net` and `build-win2012-2.adoptopenjdk.net` are bare metal machines.  
I hadn't noticed that the referenced test machine was at Packet.

**Comment by sxa555** [comment]
I also wouldn't say no to a second Windows 2012R2 for JCK testing if you're able to get one - our first one is at Joyent. It doesn't have to be anything large, 2CPU, 4GB sort of spec would likely be adequate/

**Comment by VermaSh** [comment]
I saw this on my notification feed. If the machine hasn't already been added, I would like to add it using the Jenkins Node Helper API. I have the jobs ready to be deployed, waiting on PR approvals (should be done soon). 

Here's a link to the API repo [openjdk-jenkins-helper
](https://github.com/AdoptOpenJDK/openjdk-jenkins-helper/pull/15)

**Comment by sxa555** [comment]
@VermaSh Sounds worth a shot. We haven't got the machine yet though

**Comment by smlambert** [comment]
To clarify, for this project we would be requesting 2 more Windows machines, 1 for JCK testing, 1 more for non-JCK testing (preferably 8GB/8CPU/Windows Server 2012R2).

**Comment by tellison** [comment]
I am provisioning two additional Windows machines.  We are limited to hostnames of <= 15 chars, so they are named:

1. `test-win2012-1.adoptopenjdk.net` to be known to Jenkins as `test-softlayer-x64-windows-2012r2-1` and,
1. `jck-win2012-1.adoptopenjdk.net` to be known to Jenkins as `jck-softlayer-x64-windows-2012r2-1`

Each one is 2 CPU, 8GB RAM, Windows 2012R2, 100Gb boot disk, 25 Gb data disk, 100 Mbs network link.

I will create a new issue for configuring these machines for project use.

**Comment by tellison** [comment]
@smlambert 

> I am not fussy on spec, can be small as @sxa555 mentioned, 2CPU, 4GB, or a duplicate of the test-packet-x64-windows-2012r2-1 (8GB, 8CPU running Windows Server 2012R2), this would improve test execution time, as we run some types of tests concurrently.

Let me know how you get on with the 2CPU 8GB test machine.  I can easily add more CPUs _in situ_ if you get more fussy and decide you would like them :-)

@VermaSh Please see issue #337 for configuring these new machines with help from your scripts.

I'm closing this issue as completed.  Please open a new issue if there are further machine requests.
